modules/user/src/main/Trophy.scala

```diff
@@ -6,7 +6,8 @@ case class Trophy(
     _id: String, // random
     user: String,
     kind: TrophyKind,
-    date: DateTime
+    date: DateTime,
+    url: Option[String]
```

Comment:
I don't know if there are larger consequences to adding an Option[String] to each Trophy mongo seems okay with loading existing trophies.